core-bundle/src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -2828,15 +2829,16 @@ protected function generateTree($path, $intMargin, $mount=false, $blnProtected=t
 			}
 
 			$strFileNameEncoded = StringUtil::convertEncoding(StringUtil::specialchars(basename($currentFile)), Config::get('characterSet'));
+			$iconAlt = sprintf($GLOBALS['TL_LANG']['MSC']['fileC'], strtoupper($objFile->extension));
```

Comment:
Can we find a better name here?

* `folderCP` → folder closed protected
* `folderC` → folder closed
* `fileC` → file closed 😕